requirements.txt

```diff
@@ -0,0 +1 @@
+python
```

Comment:
What is this? this file should follow its own rules. See : https://www.idkrtm.com/what-is-the-python-requirements-txt/